assemble.py

```diff
@@ -47,20 +47,26 @@ def losetup(path: str) -> str:
     # up, thus the "No such file or directory". When this happens, we need to
     # remount /dev in order to see the new device.
     try:
+        # get the next available loop device
+        loop_device = cmd('losetup', '-f', capture=True).decode().rstrip()
```

Comment:
Could you change the line below to be `cmd('losetup', '-P', loop_device, path)`?

Reply:
I think so.

Reply:
done